.editorconfig

```diff
@@ -1,2 +1,13 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace=true
+
 [*.{kt,kts}]
 max_line_length=120
+
+[*.rb]
+indent_style = space
+
+[**FastFile]
```

Comment:
```suggestion
[**Fastfile]
```

From the [docs](https://editorconfig.org/#file-format-details), these patters are case sensitive.

<img width="743" alt="Screen Shot 2020-07-13 at 4 01 59 pm" src="https://user-images.githubusercontent.com/1218433/87275830-57c04600-c522-11ea-9488-1b2bf308b452.png">


I tried to add `indent_size = 16` here, and it only worked with `[**Fastfile]`, not `[**Fastfile]`.

Reply:
Only having the first letter capitalized seems to be the norm, so I'm good to follow this here too :)